src/App.jsx

```diff
@@ -1,35 +1,41 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import React, { useState } from "react";
+import Counter, { maxChars } from "./components/Counter";
+import Feed from "./components/Feed";
+import TweetForm from "./components/TweetForm";
+import "./App.css";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [tweets, setTweets] = useState([]);
+  const [message, setMessage] = useState("");
+
+  const addTweet = (tweet) => {
+    setTweets((previousTweets) => [tweet, ...previousTweets]);
+  };
 
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
+    <div className="container">
+      <div className="header">
+        <h1>Tweeter</h1>
+        <p className="subheader">
+          If you're one of the lucky users in the dropdown menu, share your
+          random thoughts below!
         </p>
       </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  )
+      <div className="top">
+        <TweetForm
+          addTweet={addTweet}
+          message={message}
+          setMessage={setMessage}
+          maxChars={maxChars}
+        />
+        <Counter chars={message} />
+      </div>
+      <br />
+      <div className="bottom">
+        <Feed tweets={tweets} />
```

Comment:
Same comment about the div wrapping TweetForm with className top. Can we remove this extra layer in the HTML and directly apply the CSS to the top level Feed element?